### ISSUE TITLE
feature request: source-map-support

### ISSUE THREAD

**Issue by undefined-moe** [myndzi/wtfnode#39]
when using `node --enable-source-maps` , the error thrown will be like this:

```
        at Object.model (/root/Hydro/packages/hydrooj/dist/entry/common.js:180:32)
            -> /root/Hydro/packages/hydrooj/src/entry/common.ts:159:32
        at Object.load (/root/Hydro/packages/hydrooj/dist/entry/worker.js:59:20)
            -> /root/Hydro/packages/hydrooj/src/entry/worker.ts:38:11
        at processTicksAndRejections (internal/process/task_queues.js:97:5)
        at async entry (/root/Hydro/packages/hydrooj/dist/loader.js:113:20)
            -> /root/Hydro/packages/hydrooj/src/loader.ts:94:20
        at async Object.load (/root/Hydro/packages/hydrooj/dist/loader.js:269:13)
            -> /root/Hydro/packages/hydrooj/src/loader.ts:239:13
```

hope wtfnode can also support .js.map files.

**Comment by myndzi** [comment]
Try importing/requriing `source-map-support` after importing wtfnode. It can work alongside that module but doesn't require it. Example from the tests directory: https://github.com/myndzi/wtfnode/blob/master/tests/test-sourcemaps.coffee

**Comment by undefined-moe** [comment]
it doesn't work for me.

![image](https://user-images.githubusercontent.com/29992205/100544113-7a16ee00-328e-11eb-9107-b84f870e1474.png)

**Comment by myndzi** [comment]
@undefined-moe I am very sorry, I missed the follow-up and never actually addressed this!

I am about to push a version that hopefully works around the problem here. I don't want to add any dependencies or peer dependencies, so it's brittle to breaking-api changes in `source-map-support`, but I added tests to verify that it doesn't break if that happens (and also that it continues to function if that module is not present at all)

**Comment by myndzi** [comment]
Published 0.9.0, should work in that :)
